content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -144,9 +143,305 @@ dependencies {
 
 {{% /tab %}} {{< /tabpane>}}
 
-### Additional instrumentations
+### OTLP Exporter
+
+This package provides auto configurations for
+[OTLP](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/otlp)
+and
+[Logging](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/logging)
+Span Exporters.
+
+#### Exporter Properties
+
+| Feature       | Property                    | Default Value    |
+| ------------- | --------------------------- | ---------------- |
+| Otlp Exporter | otel.exporter.otlp.endpoint | `localhost:4317` |
```

Comment:
```suggestion
| OTLP Exporter | otel.exporter.otlp.endpoint | `localhost:4317` |
```

Reply:
Suggestion for this table and the others:

- Put properties in code font.
- Values should be in code font only if necessary. E.g. "true" or a number (used below) do not need to be in code font IMHO